closure/library/BUILD

```diff
@@ -27,6 +27,7 @@ closure_js_library(
         "JSC_INVALID_SUPPRESS",
         "JSC_MISSING_JSDOC",
         "JSC_MISSING_RETURN_JSDOC",
+        "JSC_MISSING_REQUIRE_CALL_WARNING",
```

Comment:
I just inspected these and something more is going on here. Each instance in the library has a suppress annotation on the function body.

[closure/goog/datasource/expr.js](https://github.com/google/closure-library/blob/master/closure/goog/datasource/expr.js#L267)
[closure/goog/graphics/vmlelement.js](https://github.com/google/closure-library/blob/master/closure/goog/graphics/vmlelement.js#L106)
[closure/goog/net/browsertestchannel.js](https://github.com/google/closure-library/blob/master/closure/goog/net/browsertestchannel.js#L349)
[closure/goog/net/channelrequest.js](https://github.com/google/closure-library/blob/master/closure/goog/net/channelrequest.js#L1122)

```
INFO: From Checking 812 JS files in //closure/library:library:
external/closure_library/closure/goog/datasource/expr.js:276: WARNING lintChecks JSC_MISSING_REQUIRE_CALL_WARNING - missing require: 'goog.ds.DataManager'
                                  goog.ds.DataManager.getInstance();
                                  ^

external/closure_library/closure/goog/graphics/vmlelement.js:116: WARNING lintChecks JSC_MISSING_REQUIRE_CALL_WARNING - missing require: 'goog.graphics.VmlGraphics'
      goog.graphics.VmlGraphics.toSizeCoord(height);
      ^

external/closure_library/closure/goog/graphics/vmlelement.js:276: WARNING lintChecks JSC_MISSING_REQUIRE_CALL_WARNING - missing require: 'goog.graphics.VmlGraphics'
  style.height = goog.graphics.VmlGraphics.toSizePx(height);
                 ^

external/closure_library/closure/goog/graphics/vmlelement.js:421: WARNING lintChecks JSC_MISSING_REQUIRE_CALL_WARNING - missing require: 'goog.graphics.VmlGraphics'
  style.height = goog.graphics.VmlGraphics.toPosPx(height);
                 ^

external/closure_library/closure/goog/net/browsertestchannel.js:376: WARNING lintChecks JSC_MISSING_REQUIRE_CALL_WARNING - missing require: 'goog.net.BrowserChannel'
      goog.net.BrowserChannel.createChannelRequest(this, this.channelDebug_);
      ^

external/closure_library/closure/goog/net/channelrequest.js:1129: WARNING lintChecks JSC_MISSING_REQUIRE_CALL_WARNING - missing require: 'goog.net.BrowserChannel'
  this.watchDogTimerId_ = goog.net.BrowserChannel.setTimeout(
                          ^

0 error(s), 6 warning(s)
```

Is it possible that JSChecker is not honoring suppress annotations on function bodies?